common/source/Util.cpp

```diff
@@ -34,6 +39,21 @@ std::string exec(const char *cmd) {
   return result;
 }
 
+int readFile(std::string fileName, int arraySize, unsigned char* array)
+{
+    //Converting Key for encryption funct
+    unsigned char keyChar[arraySize] = {};
+    //Grabbing key
+    std::ifstream keyFile(fileName, std::ios::binary);
+
+    keyFile.read(reinterpret_cast<char*>(keyChar),arraySize);
+
+    for (int lv = 0; lv < arraySize; lv++)
+    {
+        array[lv] = keyChar[lv];
+    }
+return 1;
```

Comment:
If it is always returning 1 why not make it void, or you could see if you can capture a return value from the read command.